### PR TITLE
alsa-lib: Add CPE ID for CVE tracking

### PR DESCRIPTION
**Pull request by neheb**
Also added HTTPS.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @tripolar @thess

### CODE REVIEW

File: libs/alsa-lib/Makefile [hunk 1]
```diff
@@ -13,14 +13,15 @@ PKG_RELEASE:=1
...
-		http://distfiles.gentoo.org/distfiles/
+		https://distfiles.gentoo.org/distfiles/
 
 PKG_HASH:=5f2cd274b272cae0d0d111e8a9e363f08783329157e8dd68b3de0c096de6d724
 PKG_MAINTAINER:=Ted Hess <thess@kitschensync.net>, \
 		Peter Wagner <tripolar@gmx.at>
 
 PKG_LICENSE:=LGPLv2.1 GPLv2
 PKG_LICENSE_FILES:=COPYING aserver/COPYING
+PKG_CPE_ID:=cpe:/o:alsa:alsa-lib
```
Comment (tripolar) [inline libs/alsa-lib/Makefile]: why do you use cpe:/o: and not cpe:/a: ? to be honest i didn't read about it but a grep gave me

> admin/htop/Makefile:PKG_CPE_ID:=cpe:/a:htop:htop
> admin/monit/Makefile:PKG_CPE_ID:=cpe:/a:tildeslash:monit
> libs/libxml2/Makefile:PKG_CPE_ID:=cpe:/a:xmlsoft:libxml2
> libs/tiff/Makefile:PKG_CPE_ID:=cpe:/a:libtiff:libtiff
> utils/unrar/Makefile:PKG_CPE_ID:=cpe:/a:rarlab:unrar

Comment (thess) [reply libs/alsa-lib/Makefile]: I don't think this is right either. ```cpe:/o:``` is for operating system (like linux kernel). Do you have a definitive reference URL I can check.

Comment (neheb) [reply libs/alsa-lib/Makefile]: I used this page as a reference: https://nvd.nist.gov/vuln/detail/CVE-2005-0087

The o might stand for Oval based on that page. Not sure.

Comment (thess) [reply libs/alsa-lib/Makefile]: There is no "alsa" anything in the CPE dictionary. I think you found a typo or error. Try to query the database here: https://nvd.nist.gov/products/cpe/search

**Comment by neheb** [comment]
Fair enough. Closing.

**Pull request closed without merging**